test/spark/app/.gitignore

```diff
@@ -0,0 +1,34 @@
+*.class
+*.log
+*.zip
+*.gz
+*.jpeg
+*.jpg
+*.png
+*~
+
+# sbt specific
+.settings/
+.cache/
+.history/
+.lib/
+dist/*
+target/
+lib_managed/
+src_managed/
+project/boot/
+project/plugins/project/
+.project
+.classpath
+.cache
```

Comment:
Most of these need to go one level up, relevant for *anything* SBT-ish.  Also `.project` and `.classpath`.  No idea what a file (not a suffix) `.class`  does, not sure why it's ehre either.

Reply:
eclipse - I'll remove